docs/proposals/202011_vertical_block_sharding.md

```diff
@@ -77,6 +77,7 @@ index size).
 
 * Automatically cap the size of index per block to X GB without impacting read performance.
   * Optional: CLI for vertical sharding of old blocks. 
```

Comment:
Let's remove this, let's not suggest implementation

Reply:
Done !